### PR TITLE
Clarify new unhollower assembly destination for MelonLoader versions over 0.6.0

### PR DESCRIPTION
**Pull request by gl1tch638**
Update quickstart.md

### CODE REVIEW

**Review by loukylor — CHANGES_REQUESTED** [review]

File: docs/modders/quickstart.md [hunk 1]
```diff
@@ -197,6 +197,7 @@ However, compared to Unity, you have to reference all the game and Unity assembl
 
 For games using the [Mono runtime](https://www.mono-project.com), all the game/Unity assemblies can be found in `[Game Directory]\[Game Name]_data\Managed\`.<br>
 For games using the IL2CPP runtime, all the game/Unity assemblies can be found in `[Game Directory]\MelonLoader\Managed\` (make sure you have ran the game with MelonLoader at least once!).
```
[the remark this answers is not in the document]
Comment (gl1tch638) [reply docs/modders/quickstart.md:199]: Ok I edited it in my repo
If i messed it up please notify, this is a first for me

**Review by gl1tch638 — COMMENTED** [review]
Included line break tag as requested :)

File: docs/modders/quickstart.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (loukylor) [reply docs/modders/quickstart.md:199]: I meant to add the tag to the line above the line you added. Make sure to remove the tag from your line. Also clarify that your line is only referring to Il2Cpp games.

[the remark this answers is not in the document]
Comment (gl1tch638) [reply docs/modders/quickstart.md]: Ok, i made the changes

**Comment by loukylor** [comment]
Awesome thank you!

**Pull request merged**